api/src/main/java/uk/gov/dhsc/htbhf/eligibility/service/EligibilityService.java

```diff
@@ -15,16 +16,25 @@
 public class EligibilityService {
 
     private final DWPClient dwpClient;
+    private final HMRCClient hmrcClient;
     private final BigDecimal ucMonthlyIncomeThreshold;
     private final Integer eligibilityCheckFrequencyInWeeks;
+    private final BigDecimal ctcAnnualIncomeThreshold;
+    private final EligibilityStatusCalculator statusCalculator;
 
 
-    public EligibilityService(@Value("${dwp.eligibility-check-frequency-in-weeks}") Integer eligibilityCheckFrequencyInWeeks,
+    public EligibilityService(@Value("${eligibility-check-frequency-in-weeks}") Integer eligibilityCheckFrequencyInWeeks,
```

Comment:
I'd suggest we have this configurable as a number of days rather than weeks for greater flexibility, and call it something like 'eligibility-check-period-length'. 'eligibility-check-frequency-in-weeks' sounds a bit like it's controlling how often we make checks, but here it's just controlling the length of the period in which we're looking for eligibility.

Reply:
Done.